tests/examples_arguments_syntax/line_chart_with_cumsum_faceted.py

```diff
@@ -0,0 +1,36 @@
+"""
+Faceted Line Chart with Cumulative Sum
+------------------------------
+This chart creates multiple line subcharts from the cumulative sum of a field, one for each category.
+"""
+# category: line charts
+import altair as alt
+from vega_datasets import data
+
+source = data.disasters()
+columns_sorted = ['Drought', 'Epidemic', 'Earthquake', 'Flood']
+
+alt.Chart(source).mark_line(
+    interpolate='basis' # Use linear interpolation with B-spline method
+).encode( 
+    alt.X('Year:Q', title=None, axis=alt.Axis(format='d')),
+    alt.Y('cumulative:Q', title=None),
+    alt.Color('Entity:N', legend=None)
+).properties(width=300, height=150).facet(
+    facet=alt.Facet(
+        'Entity:N',
+        title=None,
+        sort=columns_sorted,
+        header=alt.Header(labelAnchor='start', labelFontStyle='italic')
+    ),
+    title={
+        'text': ['Cumulative casualties by type of disaster', 'in the 20th century'],
+        'anchor': 'middle'
+    },
```

Comment:
```suggestion
    title=alt.Title(
        ['Cumulative casualties by type of disaster', 'in the 20th century'],
        anchor='middle'
    ),
```

Same in the other file

Reply:
- Updated title definition from dictionary to alt.Title class for consistency and readability.